bedrock/firefox/templates/firefox/whatsnew/fx54/whatsnew-54-ja-zhcn-base.html

```diff
@@ -0,0 +1,22 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/fx54/whatsnew-54.html" %}
+
+{% add_lang_files "firefox/sendto" "download" %}
```

Comment:
"download" should be removed, file doesn't exist anymore. And it should definitely load "firefox/whatsnew_50"